config.go

```diff
@@ -278,6 +278,12 @@ type Config struct {
 	// Optional: defaults to false
 	AutomaticCutover bool
 
+	// These two values configure the amount of times Ferry should attempt to
+	// retry acquiring the cutover lock, and for how long the Ferry should wait
+	// before attempting another lock acquisition
+	MaxCutoverRetries       int
+	CutoverRetryWaitSeconds int
```

Comment:
These variables should be in sharding/config.go because they don't pertain to the base Ferry.

Reply:
Is there a `ValidateConfig` method we can use to default these values in the sharding config? If not, any reason we shouldn't add one?

Reply:
We should definitely initialize default values for some config options somewhere. I don't recall off the top of my head where.

Reply:
I added another `ValidateConfig` to the sharding ferry's configuration that checks these two values and sets proper defaults and then calls the embedded struct's (ferry config) `ValidateConfig` method.